packages/frontend/src/features/semanticViewer/components/Content.tsx

```diff
@@ -102,86 +176,13 @@ const Content: FC = () => {
 
                 {!!view && <Filters />}
 
-                <RunSemanticQueryButton ml="auto" />
+                <RunSemanticQueryButton
+                    ml="auto"
+                    onClick={runSemanticLayerQuery}
```

Comment:
not important. A useful pattern that I find easy to read is `onClick` -> `handleSomething`. `handle` prefix means it's defined within the same component. If it's `onClick` -> `onClick`, then the `on` prefix tells the actual handler is defined somewhere in the parent component. Something without the `handle` or `on` prefix means it's a utility function of some kind and is not related to the React lifecycle at all.